pomegranate/base.pxd

```diff
@@ -16,15 +15,19 @@ cdef class Model( object ):
 	cdef double _log_probability( self, double symbol ) nogil
 	cdef double _mv_log_probability( self, double* symbol ) nogil
 	cdef double _vl_log_probability( self, double* symbol, int n ) nogil
-	cdef void _v_log_probability( self, double* symbol, double* log_probability, int n ) nogil
-	cdef double _summarize( self, double* items, double* weights, SIZE_t n ) nogil
+	cdef void _v_log_probability(
+		self, double* symbol, double* log_probability, int n ) nogil
```

Comment:
I'd like to follow sklearn's style, where you get as many arguments on a line as possible, and then start that line at the same indentation as the first argument. Like:

```
def test(a, b, c....
             d, e, f)
```